composer/managers.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.contrib.sites.shortcuts import get_current_site
+
+try:
+    from jmbo.managers import DefaultManager
+    from jmbo.managers import PermittedManager as JmboPermittedManager
+    has_jmbo = True
+except ImportError:
+    from django.db.models import Manager as DefaultManager
+    has_jmbo = False
+
+try:
+    from crum import get_current_request
+    has_crum = True
+except ImportError:
+    has_crum = False
+
+
+class ComposerPermittedManager(models.Manager):
+    """Get site from request if available, otherwise default."""
```

Comment:
```
"""
Get site from request if available, otherwise default.
"""
or
"""Get site from request if available, otherwise default.
"""
```
Are the preferred PEP-8 ways of writing doc strings.